modules/calcite/src/test/sql/types/list/list_aggregates.test_ignore

```diff
@@ -1,29 +1,26 @@
 # name: test/sql/types/list/list_aggregates.test
 # description: Test lists with aggregations
 # group: [list]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15563
-
-statement ok
-PRAGMA enable_verification
+# Ignore https://issues.apache.org/jira/browse/IGNITE-16417
 
 query II
-select min(i::varchar), max(i::varchar) from range(10) tbl(i);
+select min(i::varchar), max(i::varchar) from table(system_range(0, 10)) tbl(i);
```

Comment:
The same here and below, `range(10)` is equivalent to `table(system_range(0, 9)`